src/main/frontend/components/ReviewTile.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+
+const ReviewTile = props => {
+  const { title, imgUrl, rating, createdAt, updatedAt } = props.review
+  const ts = new Date(createdAt);
+  const year = ts.getFullYear();
+  const month = ts.getMonth();
+  const day = ts.getDay();
+  const hour = ts.getHours();
+  const minutes = ts.getMinutes();
+  return (
+      <div>
+        <img src={imgUrl} width="300px" />
+          <h1>{title}</h1>
+          <p>Rating: {rating}</p>
+          <p>CreatedAt: {month}/{day}/{year} </p>
+      </div>
+  )
```

Comment:
14 -16 can be on the same line as 13 since they are all child elements of the same div.  The `div` itself looks to be indented 4 spaces

Reply:
Tried to jam it all on one line but Intellij auto format had another idea. Squeezed it as much as I could bound auto format.